cocos/renderer/gfx-gles2/GLES2Commands.cc

```diff
@@ -1395,6 +1395,17 @@ void GLES2CmdFuncExecuteCmds(GLES2Device* device, GLES2CmdPackage* cmd_package)
         GLES2CmdBindStates* cmd = cmd_package->bindStatesCmds[cmd_idx];
         is_shader_changed = false;
           
+          if(cmd->viewportDirty)
+          {
+              glViewport(cmd->viewport.left, cmd->viewport.top, cmd->viewport.width, cmd->viewport.height);
+              cache->viewport = cmd->viewport;
+          }
```

Comment:
If use like this, then does `cache` need to record viewport?

Reply:
begin render pass will use cache to setup viewport